CharacterGenerator/CharacterGenerator/CharacterTableViewController.m

```diff
@@ -0,0 +1,144 @@
+//
+//  CharacterTableViewController.m
+//  CharacterGenerator
+//
+//  Created by Mesfin Bekele Mekonnen on 8/6/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CharacterTableViewController.h"
+#import "Bender.h"
+#import "BenderManager.h"
+
+@interface CharacterTableViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *nameTextField;
+@property (nonatomic, strong) IBOutletCollection(UIButton) NSArray *locationQuestionButtons;
+@property (nonatomic, strong) IBOutletCollection(UIButton) NSArray *textureQuestionButtons;
+@property (nonatomic, strong) IBOutletCollection(UIButton) NSArray *colorQuestionButtons;
+
+@property (nonatomic) NSString *location;
+@property (nonatomic) NSString *color;
+@property (nonatomic) NSString *texture;
+
+
+@end
+
+@implementation CharacterTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    UITapGestureRecognizer *tap = [[UITapGestureRecognizer alloc]
+                                   initWithTarget:self
+                                   action:@selector(dismissKeyboard)];
+    [self.view addGestureRecognizer:tap];
+    
+}
+
+-(BOOL)fieldsComplete{
```

Comment:
This method would be improved by documentation. What constitutes completeness? Why does `if(i==self._xyzQuestionButtons_.count-1)` result in a NO?